data/mainnet/tokens.js

```diff
@@ -9,6 +9,8 @@ const {
 } = require('../constants')
 const addresses = require('./addresses.json')
 
+const libAddresses = require('../../src/lib/data/addresses.json')
```

Comment:
I think just use the `data/mainnet/addresses.json` file here. If you need an address, add it there.